src/helpers.rs

```diff
@@ -895,3 +895,7 @@ impl std::fmt::Display for HexRange {
         write!(f, "[{:#x}..{:#x}]", self.0.start.bytes(), self.0.end().bytes())
     }
 }
+
```

Comment:
Should I consider adding `assert_target_os_is_unix` as well ? 
If yes, that would take care of snippets like 
```rs
        let target_os = &this.tcx.sess.target.os;
        assert!(
            target_os == "linux" || target_os == "macos",
            "`getenv` is only available for the UNIX target family"
        );
```

Reply:
Yes please do. :)